build.rs

```diff
@@ -42,7 +42,7 @@ fn main() {
         println!("cargo:rerun-if-changed={}", hdr.display());
     }
 
-    process_linker_file(&outdir);
+    //process_linker_file(&outdir);
```

Comment:
Is this not needed anymore? If so, could you please remove it?

Reply:
I thought that original linker files are no longer required to build, however, for examples besides blinky there is some linkage issues that I will figure soon hopefully. If it turns out that original linker files were required, I'll uncomment this.